flexbox1/flexbox1.css

```diff
@@ -0,0 +1,84 @@
+
+*{
+    font-family: "Arial";
+}
+
+
+.big-container{
+    display: flex;
+    flex-direction: column;
+    gap: 0px;
+    justify-content: center;
+    align-items: center;
+}
+
+.subtitle{
+    font-weight: lighter;
+    color: gray;
+    margin-top: 0px;
+    text-align: center;
+}
+
+.container{
+    width: fit-content;
```

Comment:
Nice!

O mica observatie ar fi ca unele proprietati css nu sunt 100% suportate de toate browserele. In cazul de fata e posibil ca `fit-content` sa nu funtioneze daca cineva vede pagina web pe Internet Explorer (https://caniuse.com/?search=fit-content)